esp/platform/txsummary.hpp

```diff
@@ -27,6 +28,10 @@
 #include <list>
 #include <map>
 
+#define DEFAULT_GROUP TXSUMMARY_GRP_CORE
```

Comment:
If DEFAULT_GROUP, why not DEFAULT_STYLE? Or DEFAULT_LOGLEVEL (yes, unrelated to your PR)?

Reply:
I'd added it originally when troubleshooting/debugging and toggling between groups. I'll remove it now.